documentation/source/contribute/contribute_openebs_doc.rst

```diff
@@ -0,0 +1,73 @@
+
+Contributing to the OpenEBS Documentation
+=========================================
+
+You can access the latest documents at http://openebs.readthedocs.io/en/latest/. 
+
+Automated builds are setup for OpenEBS documentation at https://readthedocs.org/projects/openebs/
+
+Sphinx is used for building the OpenEBS documentation - http://www.sphinx-doc.org/en/stable/tutorial.html
+
+Documentation Contributors
+--------------------------
+
+1. If you would like to give feedback on existing content, create an issue on documentation (see, `Creating an Issue`_).
+
+2. If you would like to contribute to new content, 
+     -  create an issue (see, `Creating an Issue`_), 
+     -  create your own branch (see, `Creating a Branch`_), 
+     -  work on the content by creating an RST file, and 
+     -  create a pull request (see, `Creating a Pull Request`_).
+
+Internal Contributors
+---------------------
+
+1. OpenEBS Lead receives documentation issues/pull requests raised by Documentation/Internal contributors. 
+2. Lead will tag issues to relevant labels which start with the name **documentation** and assign             reviewers and approvers such as feature owner, doc person, and approver. 
+3. The assignee works on the issue either developing content in RST files or editing the content. 
+4. The issue owner sends the content either as a  rst/md file, or as a comment in the issue. 
```

Comment:
"issue owner" -- could be the assignee itself. So we should probably re-word this step as, 

The assignee can get the content from other collaborators of the OpenEBS project either as rst/md file or as a comment in the issue. The assignee can also use the "OpenEBS slack" channel to collection additional information from the community.

Reply:
Done